MUXSDKStats/MUXSDKStats/MUXSDKPlayerBinding.m

```diff
@@ -109,6 +109,14 @@ - (void)attachAVPlayer:(AVPlayer *)player {
     [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(handleAVPlayerError:) name:AVPlayerItemNewErrorLogEntryNotification object:nil];
     [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(handleConnectionTypeDetected:) name:@"com.mux.connection-type-detected" object:nil];
     
+    //
+    // dylanjhaveri
+    // See MUXSDKConnection.m for the tvos shortcoming
+    //
+    if ([[UIDevice currentDevice] userInterfaceIdiom] != UIUserInterfaceIdiomTV) {
```

Comment:
i think you mentioned that this should be changed to an OS check instead?

Reply:
Not exactly, what I said is that the code block in the `initSDK` method is defining 2 variables deviceCategory and osFamily, and the deviceCategory was being used instead of osFamily to do the check, so osFamily should be the one used, but it can really be simplified to this check, basically detect the connection if the device is not an Apple TV.